webapp/src/component/EmailNotVerifiedView.tsx

```diff
@@ -78,37 +117,39 @@ export const EmailNotVerifiedView = () => {
         >
           <StyledContainer>
             <StyledHeader variant="h4">
-              <T keyName="verify_email_title" />
+              <T keyName="verify_email_check_inbox" />
             </StyledHeader>
             <StyledDescription variant="body1" mb={2}>
               <T
-                keyName="verify_email_description"
-                params={{ email: email, b: <StyledEnabled /> }}
+                keyName="verify_email_we_sent_email"
+                params={{ email: email, b: <BoldSpan /> }}
               />
             </StyledDescription>
             <StyledImg src={imageSrc} alt="Verify email" />
-            <StyledHint variant="body2">
-              <T keyName="verify_email_didnt_receive_email_hint" />
-            </StyledHint>
-            <Button
-              variant="contained"
-              data-cy="resend-email-button"
-              onClick={() =>
-                resendEmail.mutate(
-                  {},
-                  {
-                    onSuccess: () => {
-                      messageService.success(
-                        <T keyName="verify_email_resend_message" />
-                      );
-                    },
-                  }
-                )
-              }
-              color="primary"
-            >
-              <T keyName="verify_email_resend_button" />
-            </Button>
+
+            <StyledDescription variant="body1">
+              {enabled ? (
+                <T
+                  keyName="verify_email_resend_link_retry_after"
+                  params={{
+                    seconds: remainingSeconds,
+                    link: <StyledLink className="disabled"></StyledLink>,
+                  }}
+                />
+              ) : (
+                <T
+                  keyName="verify_email_resend_link"
+                  params={{
+                    link: (
+                      <StyledLink
+                        data-cy="resend-email-button"
+                        onClick={handleResendEmail}
```

Comment:
User can still click this when the request is pending. Would be nice to disable this when `resendEmail.isLoading`.